docs/development/releasing.rst

```diff
@@ -115,7 +115,12 @@ packages that use the full bugfix/maintenance branch approach.)
 
    Don't forget to remove any non-committed files with::
 
-      $  git clean -dfx
+      $ git clean -dfx
+
+#. Make sure the source distribution doesn't inherit limited permissions
+   following you default umask::
+
+     $ umask 0022
```

Comment:
Originally I've changed the local file permissions with chmod but that has no effect on the release content. Then did umask at this point, and it seemed to be a working solution. My understanding was that the tarball files indeed inherited their permission from the actual umask rather than the permissions in the dev version.

I'm inclined to merge this as is, and if there are issues revisit after the new release (I plan to do a final 1.3.3 soon, before feature freeze for 2.0)